spec/models/social_networking/profile_answer_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "spec_helper"
+
+module SocialNetworking
+  describe ProfileAnswer, type: :model do
```

Comment:
I like the idea of using `existing_answer = ProfileAnswer.first`.  I am not too concerned about the 15 lines considering some are b/c of rubocop, the `it`, and the `end`s.  Let me update this.  Thanks.